### PR TITLE
Add pdf-view-restore to pdf layer

### PR DESCRIPTION
**Pull request by dalanicolai**
I think restore last visited page should probably be the default in a pdf-viewer. [pdf-view-restore](https://github.com/007kevin/pdf-view-restore) does it simple and fine.

### CODE REVIEW

File: layers/+readers/pdf/packages.el [hunk 1]
```diff
@@ -118,3 +119,9 @@
         "r"              'pdf-occur-revert-buffer-with-args
         "*"              'spacemacs/enter-ahs-forward
         "?"              'evil-search-backward))))
+
+(defun pdf/init-pdf-view-restore ()
+  (use-package pdf-view-restore
+    :after pdf-tools
```
Comment (smile13241324) [inline layers/+readers/pdf/packages.el:125]: Please lazy load.

File: layers/+readers/pdf/packages.el [hunk 2]
```diff
@@ -118,3 +119,9 @@
...
+    :config
+    (add-hook 'pdf-view-mode-hook 'pdf-view-restore-mode)))
```
Comment (smile13241324) [inline layers/+readers/pdf/packages.el:127]: Hooks should go into the init section to trigger lazy loading.

**Review by smile13241324 — CHANGES_REQUESTED** [review]
Nice addition to the pdf layer. However I think you need to double check the loading of your package.

File: layers/+readers/pdf/packages.el (hunk 2 above)
Comment (dalanicolai) [reply layers/+readers/pdf/packages.el:127]: Ai! I should have learned that by now... :grin:

**Review by smile13241324 — APPROVED** [review]
Looks fine now, will be merged soon.

**Pull request merged**